LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/html/browsers/browsing-the-web/navigating-across-documents/failure-check-sequence.https-expected.txt

```diff
@@ -0,0 +1,6 @@
+
+PASS CSP check precedes X-Frame-Options check
+PASS CSP check precedes COEP check - CSP header first
+PASS CSP check precedes COEP check - COEP header first
+FAIL COEP check precedes X-Frame-Options check assert_equals: expected (string) "coep" but got (object) null
```

Comment:
Interesting. I wonder what this bug is due to.

Reply:
This is WK1, no COEP support.